doc/source/user_guide/timeseries.rst

```diff
@@ -2337,7 +2337,7 @@ Time zone handling
 ------------------
 
 pandas provides rich support for working with timestamps in different time
-zones using the ``pytz`` and ``dateutil`` libraries or :class:`datetime.timezone`
+zones using the ``zoneinfo``, ``pytz`` and ``dateutil`` libraries or :class:`datetime.timezone`
```

Comment:
Do we even really need to advertise pytz any more or can it be fully replaced by zoneinfo?

Reply:
I _think_ zoneinfo objects support all operations that `pytz` does. I was planning on keeping it around since it will be supported via an optional dependency I suppose.

If we ever deprecate pytz support, I would fully support not advertising it anymore